### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by vtieto**

### CODE REVIEW

File: brecker.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Michael Brecker
```
Comment (stitt1977) [inline brecker.md]: Can we make this title all-caps?

File: brecker.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (stitt1977) [inline brecker.md:5]: You might want to mention that Michael recorded with his brother before joining Dreams.

File: brecker.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+Most of Brecker's early work is marked by an approach informed as much by rock guitar as by R&B saxophone.[citation needed] After Dreams, he worked with Horace Silver and then Billy Cobham before once again teaming up with his brother Randy to form the Brecker Brothers. The band followed jazz-funk trends of the time, but with more attention to structured arrangements, a heavier backbeat, and a stronger rock influence.[citation needed] The band stayed together from 1975 to 1982, with consistent success and musicality. In 1977 he founded the Seventh Avenue South jazz club with his brother Randy.[6]
+
```
Comment (stitt1977) [inline brecker.md:7]: Maybe a good idea to add which horns Michael played.

**Review by stitt1977 — CHANGES_REQUESTED** [review]
_This is a test._

File: brecker.md (hunk 1 above)
Comment (vtieto) [reply brecker.md]: Yes, good idea!

File: brecker.md (hunk 2 above)
Comment (stitt1977) [reply brecker.md:5]: Okay, looks good.

File: brecker.md (hunk 3 above)
Comment (stitt1977) [reply brecker.md:7]: LGTM

**Review by stitt1977 — APPROVED** [review]

**Comment by vtieto** [comment]
Merging.

**Pull request merged**